src/views/AppView/AppView.tsx

```diff
@@ -1,28 +1,31 @@
 import React, { FunctionComponent, useState } from "react";
-import { Faction, RuleSet } from "../../types";
+import { Faction, RuleSet, Model } from "../../types";
 import AddUnitsView from "../AddUnitsView/AddUnitsView";
 import AddUnitsZeroStateView from "../AddUnitsZeroStateView/AddUnitsZeroStateView";
 import ChooseFactionView from "../ChooseFactionView/ChooseFactionView";
 import ChooseRuleSetView from "../ChooseRuleSetView/ChooseRuleSetView";
+import SetUnitView from "../SetUnitView/SetUnitView";
 import StartScreenView from "../StartScreenView/StartScreenView";
 import ROUTES, { INITIAL_ROUTE } from "./routes";
 
 interface State {
   ruleSet: RuleSet | null;
   faction: Faction | null;
   warbandTotal: number;
+  models: Model[];
 }
 
 const initialState: State = {
   ruleSet: null,
   faction: null,
-  warbandTotal: 0
+  warbandTotal: 0,
+  models: []
 };
 
 const AppView: FunctionComponent = () => {
   const [currentRoute, setCurrentRoute] = useState(INITIAL_ROUTE);
   const [state, setState] = useState<State>(initialState);
-
+  console.log(state);
```

Comment:
We should remove this console.log statement... even if it was probably me who added it